Testing/integration/VM/setup.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+set -xeuo pipefail
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+read -p "About to run visudo, set %admin ALL = (ALL) NOPASSWD: ALL"
+sudo visudo
+
+# Unpack and install xcode
+if [ ! -e /Applications/Xcode.app ]; then
+	(cd /Applications && xip --expand ${SCRIPT_DIR}/Xcode*.xip)
+	sudo xcode-select -s /Applications/Xcode.app
+	sudo xcodebuild -license accept
+	# ... and open it to fixup stuff
+	open /Applications/Xcode.app
+fi
+
+# Install brew
+if ! command -v brew &> /dev/null; then
+	/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

Comment:
Might be better to install from a known release instead of `HEAD`

Reply:
Doesn't look like this installer script actually has any releases, nor does it provide a way to easily pin a specific version of brew to be installed :(